### ISSUE TITLE
Selenium Test not running, chrome version

### ISSUE THREAD

**Issue by jfsanchezaguilar** [actions/virtual-environments#2167]
Description
Selenium test fail when run on Windows Server 2019 image

Area for Triage:
Win Server 2019

Question, Bug, or Feature?:
Bug

Virtual environments affected

- [ ] Ubuntu 16.04
- [ ] Ubuntu 18.04
- [ ] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11.0
- [ ] Windows Server 2016 R2
- [x] Windows Server 2019

Expected behavior
Visual Studio Selenium test on Chrome should run.

Actual behavior
The selenium test don't run, Chrome Driver installed 87.0.4280.20 but the Chrome browser installed is 86.0.4240.198. 
Error shown:
> Error: System.InvalidOperationException: session not created: This version of ChromeDriver only supports Chrome version 87
> Current browser version is 86.0.4240.198 with binary path C:\Program Files\Google\Chrome\Application\chrome.exe (SessionNotCreated)

**Comment by maxim-lobanov** [comment]
@jfsanchezaguilar , looks like you update ChromeDriver in runtime.

Both Chrome and ChromeDriver are 86.x on 20201116.1 image. See details in https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#browsers-and-webdrivers

**Comment by jfsanchezaguilar** [comment]
@maxim-lobanov you are right, one of the developers added the nuget for the chrome driver, so this cause the test to use the latest version for the chrome driver. Sorry for the inconvenience.

**Comment by maxim-lobanov** [comment]
Sure, no problem

**Comment by web-padawan** [comment]
@maxim-lobanov Hi, is there any ETA for updating to 87.x?

We would like to use `selenium-stadnalone` 6.22.2 which has a fix for Edge driver incorrectly downloaded on Linux
(https://github.com/vvo/selenium-standalone/issues/515). Currently we observe this bug in GitHub Actions:

```
     Error: Missing /home/runner/work/web/web/node_modules/selenium-standalone/.selenium/chromiumedgedriver/latest-x64-msedgedriver
      at /home/runner/work/web/web/node_modules/selenium-standalone/lib/check-paths-existence.js:16:29
      at suppressedCallback (fs.js:215:5)
      at FSReqCallback.oncomplete (fs.js:156:23)
```

However, the version with a fix already uses 87.0.4280.20.

So we can't run tests using older `selenium-standalone` version locally (the ChromeDriver that it uses fails with Chrome 87). But we can't update it either because of the fact that it breaks CI. Is there anything we could do about it?

**Comment by maxim-lobanov** [comment]
@web-padawan , approximately end of this week.
